File: Homework/Homework/Calculator.cs
namespace HomeWork
{
    public static class Calculator
    {
        public static int Calculate(int val1, CalculatorOperation operation, int val2)
        {
            var result = operation switch
            {
                CalculatorOperation.Plus => val1 + val2,
                CalculatorOperation.Multiply => val1 * val2,
                CalculatorOperation.Minus => val1 - val2,
                _ => val1 / val2,
            };
            
            return result;
        }
    }
}

File: Homework5/Calculator.fs
module Homework5.Calculator
let inline Calculate (val1, operation: CalculatorOperation, val2) =
    match operation with
    | CalculatorOperation.Plus -> val1 + val2
    | CalculatorOperation.Multiply -> val1 * val2
    | CalculatorOperation.Minus -> val1 - val2
    | _ -> val1 / val2
